Makefile

```diff
@@ -121,15 +126,14 @@ install:
 	$(INSTALL) -d $(DESTDIR)/usr/models/yang/
 	$(INSTALL) -D $(TOPDIR)/models/yang/sonic/*.yang $(DESTDIR)/usr/models/yang/
 	$(INSTALL) -D $(TOPDIR)/models/yang/sonic/common/*.yang $(DESTDIR)/usr/models/yang/
-	$(INSTALL) -D $(TOPDIR)/src/cvl/schema/*.yin $(DESTDIR)/usr/sbin/schema/
-	$(INSTALL) -D $(TOPDIR)/src/cvl/testdata/schema/*.yin $(DESTDIR)/usr/sbin/schema/
```

Comment:
YIN files under testdata/schema/ creating any issue ? Why are we removing this line ? CVL UT will fail for this. We have plan to package test schema and copy in test DUT conditionally.

Reply:
This change is coming through https://github.com/project-arlo/sonic-mgmt-framework/pull/259/files

All the yin files are moved to build/cvl directory and are being copied in the package.